source/api_documentation/transactions_api/_affiliate_param_table.rst

```diff
@@ -35,15 +35,15 @@
 
   * - affiliate_payout_localized
     - Earnings
-    - Amount paid out to the affiliate
+    - Amount paid out to the affiliate |rollup_summed_total_message|
 
   * - call_result_description_detail
     - Call Result
-    - Status of the transaction
+    - Status of the transaction |rollup_affects_call_message|
 
   * - call_source_description
     - Source
-    - Source of the transaction
+    - Source of the transaction |rollup_affects_call_message|
```

Comment:
Was call_source affected? This just indicated where the call came from right?

Reply:
Good point. Thank you for pointing this out. I'll remove this.